en/5-roadmap.md

```diff
@@ -65,23 +62,22 @@ Our roadmap would follow the priority order below:
 - Enable Deeds renting
 - Xeed creation
 - Badges improvement and credentials development
-- DeFi services – in the wallet, credit cards partnership
+- DeFi services – in the wallet, credit card partnership
 
-We are counting on the early adopters from the DAOs and tech-savvy companies during the building stage. We should also have some early adopters from traditional companies that gain immediate value from the WoM by providing relevant engagement programs to their people, on top of their current traditional compensation plans.
+During the building stage, we are counting on the early adopters from the DAOs and tech-savvy companies. We should also have some early adopters from traditional companies that gain immediate value from the WoM by providing relevant engagement programs to their people on top of their current conventional compensation plans.
```

Comment:
During the building stage, we rely on early adopters from DAOs and tech-savvy companies. Additionally, we anticipate early adopters from traditional companies benefiting immediately from the WoM. They can enhance engagement programs for their employees alongside existing compensation plans.